src/js/content.js

```diff
@@ -1,10 +1,13 @@
 /* Detect whether recipe is present on page */
+async function load(url) {
+  return await import(chrome.extension.getURL(url));
```

Comment:
Let's use `browser` instead of `chrome` to not break Firefox compatibility.

Also MDN suggests `browser.extension.getURL()` is deprecated in favor of `browser.runtime.getURL()`

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extension/getURL